### DISCUSSION TITLE
Import trace for requested module nextjs

### DISCUSSION THREAD

**Discussion by rokyuddin** [vercel/next.js#60510] [App Router]
Import trace for requested module:
../../.local/share/pnpm/global/5/.pnpm/debug@4.1.1/node_modules/debug/src/node.js
../../.local/share/pnpm/global/5/.pnpm/debug@4.1.1/node_modules/debug/src/index.js
./node_modules/.pnpm/follow-redirects@1.15.4/node_modules/follow-redirects/debug.js
./node_modules/.pnpm/follow-redirects@1.15.4/node_modules/follow-redirects/index.js
./node_modules/.pnpm/axios@1.6.5/node_modules/axios/lib/adapters/http.js
./node_modules/.pnpm/axios@1.6.5/node_modules/axios/lib/adapters/adapters.js
./node_modules/.pnpm/axios@1.6.5/node_modules/axios/lib/axios.js
./node_modules/.pnpm/axios@1.6.5/node_modules/axios/index.js
./src/lib/api.ts
./src/hooks/reactQuery/brandQuery.ts
./src/app/brands/ShowBrands.tsx
 ⚠ ../../.local/share/pnpm/global/5/.pnpm/debug@4.1.1/node_modules/debug/src/node.js
Module not found: Can't resolve 'supports-color' in '/home/miicon/.local/share/pnpm/global/5/.pnpm/debug@4.1.1/node_modules/debug/src'

**Comment by icyJoseph** [comment]
It'd seem that this is a problem with one of axios' dependencies, called  follow-redirects which in turn has a dependency on debug. 

This is probably caused by a mix of client/server side code, but if its just a warning I think you can ignore it for now.
